gips/data/aod/aod.py

```diff
@@ -206,7 +256,12 @@ def process_mean(cls, filenames, fout):
             totalvar[inds] = numpy.divide(totalvar[inds], counts[inds])
             imgout[1].Write(totalvar)
             t = datetime.datetime.now() - start
-            VerboseOut('%s: mean/var for %s files processed in %s' % (os.path.basename(fout), len(filenames), t))
+            utils.verbose_out(
+                '%s: mean/var for %s files processed in %s' %
+                (os.path.basename(fout), len(filenames), t)
+            )
+        else:
+            raise Exception('No filenames provided')
```

Comment:
optional:  Raise at the top of the method if there are no filenames, then the rest of the method can be moved out of the conditional.  However this keeps the diff smaller, which has value too.